xarray/backends/common.py

```diff
@@ -143,7 +146,11 @@ def add(self, source, target):
             self.sources.append(source)
             self.targets.append(target)
         else:
-            target[...] = source
+            try:
+                target[...] = source
+            except TypeError:
+                # workaround for GH: scipy/scipy#6880
+                target[slice(None, None, None)] = source
```

Comment:
@shoyer  - I raised the `Ellipsis` issue with the scipy folks: https://github.com/scipy/scipy/issues/6880. This seems to work although maybe it isn't the most robust solution, I don't know.